### ISSUE TITLE
Error building CodecZlib (not #38)

### ISSUE THREAD

**Issue by nickeubank** [bicycle1885/CodecZlib.jl#42]
Any chance anyone knows what this is?

```
julia> Pkg.build("CodecZlib")
  Building CodecZlib → `~/.julia/packages/CodecZlib/9jDi1/deps/build.log`
┌ Error: Error building `CodecZlib`: 
│ [ Info: Downloading https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.4/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz to /Users/Nick/.julia/packages/CodecZlib/9jDi1/deps/usr/downloads/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz...
│ ERROR: LoadError: Could not download https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.4/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz to /Users/Nick/.julia/packages/CodecZlib/9jDi1/deps/usr/downloads/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz:
│ ErrorException("")
│ Stacktrace:
│  [1] error(::String) at ./error.jl:33
│  [2] #download#89(::Bool, ::Function, ::String, ::String) at /Users/Nick/.julia/packages/BinaryProvider/4F5Hq/src/PlatformEngines.jl:498
│  [3] #download at ./none:0 [inlined]
│  [4] #download_verify#90(::Bool, ::Bool, ::Bool, ::Function, ::String, ::String, ::String) at /Users/Nick/.julia/packages/BinaryProvider/4F5Hq/src/PlatformEngines.jl:567
│  [5] #download_verify at ./none:0 [inlined]
│  [6] #install#129(::Prefix, ::String, ::Bool, ::Bool, ::Bool, ::Function, ::String, ::String) at /Users/Nick/.julia/packages/BinaryProvider/4F5Hq/src/Prefix.jl:314
│  [7] (::getfield(BinaryProvider, Symbol("#kw##install")))(::NamedTuple{(:prefix, :force, :verbose),Tuple{Prefix,Bool,Bool}}, ::typeof(install), ::String, ::String) at ./none:0
│  [8] top-level scope at /Users/Nick/.julia/packages/CodecZlib/9jDi1/deps/build.jl:89
│  [9] include at ./boot.jl:317 [inlined]
│  [10] include_relative(::Module, ::String) at ./loading.jl:1044
│  [11] include(::Module, ::String) at ./sysimg.jl:29
│  [12] include(::String) at ./client.jl:392
│  [13] top-level scope at none:0
│ in expression starting at /Users/Nick/.julia/packages/CodecZlib/9jDi1/deps/build.jl:78
│ [17:59:44] dyld: Library not loaded: @rpath/libssl.1.0.0.dylib
│ [17:59:44]   Referenced from: /Users/Nick/anaconda/lib/libssh2.1.dylib
│ [17:59:44]   Reason: image not found
└ @ Pkg.Operations /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1097
```

**Comment by nickeubank** [comment]
Nope, different error but same issues as #38 -- anacaonda's curl was broken!

**Comment by nickeubank** [comment]
will be fixed by  #30956
